gluster_peers_bricks.yml

```diff
@@ -0,0 +1,12 @@
+---
+#
+# Create gluster trusted storage pool and bricks
+# on all available devices on gluster nodes.
+#
+
+- hosts: gluster
+  remote_user: root
+  roles:
+  #- { role: qe-gluster-peers, gluster_nodes: "{{groups['gluster']}}"}
```

Comment:
Checking on my usm5 cluster (4 storage servers), I see that running role `cluster-centos` role again (so that the status is ok) takes just 2.73s, so it should not be slowing the setup down.